### ISSUE TITLE
Sync actions fail with error on an undefined field

### ISSUE THREAD

**Issue by kamaz** [commercetools/nodejs#1545]
### Description

I was wondering what is expected behaviour from commercetools nodejs sdk perspective in the following scenario:

1. Create a customer: 
```json
{
  "key": "abc",
  "email": "abc@test.com",
  "password": "password"
}
```

2. Use sync actions using the same payload:
```json
{
  "key": "abc",
  "email": "abc@test.com",
  "password": "password"
}
```

```js
var your => (code) => here;
```

**Comment by ashishhk** [comment]
Hey @kamaz, please go ahead with creating PR, I hope its still applicable, we would be happy to help you.

**Comment by kamaz** [comment]
@ashishhk yes that is still applicable I was wondering which route would you like to take: 

1. skipping a property 
2. setting to an `empty` array

**Comment by MarkusAzer** [comment]
Thanks @kamaz for reporting this 👍 
well, we had same issue for categories and this PR solved it [1505](https://github.com/commercetools/nodejs/pull/1505).

Basically, we follow second suggestion that you suggested by using this function `copyEmptyArrayProps` which changed lately to make it more robust [1532](https://github.com/commercetools/nodejs/pull/1532)

I believe that we just need to add `copyEmptyArrayProps` to `createBuildActions` for `customer`
like here [1505](https://github.com/commercetools/nodejs/pull/1505/files#diff-68530559af2489b6045662f62b428866R87)
